### PR TITLE
Add Null static field to handles

### LINKED ISSUE #639: Null value for HINSTANCE
**Is your feature request related to a problem? Please describe.**
Inside a method that returns a HINSTANCE, the returning null value has to be explicitly cast to an HINSTANCE: `(HINSTANCE)0` .

``` C#
public unsafe static HINSTANCE LoadLibraryFromSystemPathIfAvailable(string libraryName)
{
   if (GetModuleHandle(Libraries.Kernel32) == 0)
   {
       return (HINSTANCE)0;
   }

    // LOAD_LIBRARY_SEARCH_SYSTEM32 was introduced in KB2533623. Check for its presence
    // to preserve compat with Windows 7 SP1 without this patch.
    fixed (char* lpLibFileName = libraryName)
    {
        HINSTANCE result = LoadLibraryEx(lpLibFileName, (HANDLE)0, LOAD_LIBRARY_FLAGS.LOAD_LIBRARY_SEARCH_SYSTEM32);
        if (result != 0)
        {
             return result;
        }

        // Load without this flag.
        if (Marshal.GetLastWin32Error() != ERROR.INVALID_PARAMETER)
        {
             return (HINSTANCE)0;
        }

        return LoadLibraryEx(lpLibFileName, (HANDLE)0, 0);
    }
}
```

**Describe the solution you'd like**
A null value for HINSTANCE, such as:

``` C#
HINSTANCE.Null
```

### PR DESCRIPTION
**Pull request by elachlan**
Fixes #639

### CODE REVIEW

**Comment by elachlan** [comment]
@fernandanavamoya

File: src/Microsoft.Windows.CsWin32/Generator.cs [hunk 1]
```diff
@@ -3885,6 +3885,12 @@ private IEnumerable<MemberDeclarationSyntax> CreateCommonTypeDefMembers(Identifi
         // If this typedef struct represents a pointer, add an IsNull property.
         if (fieldType is IdentifierNameSyntax { Identifier: { Value: nameof(IntPtr) or nameof(UIntPtr) } })
         {
+            // public static readonly IntPtr Null;
```
Comment (jnm2) [inline src/Microsoft.Windows.CsWin32/Generator.cs:3888]: This comment is now misleading.

File: src/Microsoft.Windows.CsWin32/Generator.cs [hunk 2]
```diff
@@ -3885,6 +3885,12 @@ private IEnumerable<MemberDeclarationSyntax> CreateCommonTypeDefMembers(Identifi
...
+            yield return FieldDeclaration(VariableDeclaration(structName.WithTrailingTrivia(TriviaList(Space)))
+                .AddVariables(VariableDeclarator(IdentifierName("Null").Identifier)))
+                .AddModifiers(TokenWithSpace(SyntaxKind.PublicKeyword), TokenWithSpace(SyntaxKind.StaticKeyword), TokenWithSpace(SyntaxKind.ReadOnlyKeyword)) // operators MUST be public
```
Comment (jnm2) [inline src/Microsoft.Windows.CsWin32/Generator.cs:3891]: There's a comment talking about operator declarations on code for a field declaration.

**Comment by AArnott** [comment]
Thanks for contributing, @elachlan!

**Pull request merged**